analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -231,6 +231,19 @@ class Cargo(
         return null
     }
 
+    /**
+     * Extract information about authors from the given [node] with package metadata.
+     */
+    private fun parseAuthors(node: JsonNode?): SortedSet<String> =
+        node?.mapNotNullTo(sortedSetOf()) { authorNode -> parseAuthorString(authorNode.textValue()) } ?: sortedSetOf()
+
+    /**
+     * Parse an author string of the form "author <email>". Extract the author name or return *null* if this is
+     * not possible.
+     */
+    private fun parseAuthorString(author: String?): String? =
+        author?.substringBefore('<')?.trim()?.ifEmpty { null }
```

Comment:
Given how often we have this function now, how about adding it as a top-level function to `PackageManager.kt`?

Reply:
Yes, this would probably make sense. Maybe in a separate PR?

Reply:
Or in a new follow-up commit in this PR. Your choice.

Reply:
`PackageManager.kt` now offers a central function for parsing author strings. The concrete package managers have been adjusted to use the new function if applicable in separate commits.